contracts/anchors/bridged/AnchorPoseidon2.sol

```diff
@@ -94,8 +96,9 @@ abstract contract AnchorPoseidon2 is MerkleTreePoseidon, ReentrancyGuard {
     uint32 insertedIndex = _insert(_commitment);
     commitments[_commitment] = true;
     _processDeposit();
+    depositNonce++;
+    emit Deposit(_commitment, insertedIndex, depositNonce, block.timestamp);
```

Comment:
Any reason you got rid of the change we discussed? I thought we had covered that insertedIndex and depositNonce are one in the same.